mirdata/datasets/acousticbrainz_genre.py

```diff
@@ -383,8 +383,7 @@ def load_extractor(path):
 
 @core.docstring_inherit(core.Dataset)
 class Dataset(core.Dataset):
-    """The acousticbrainz genre dataset
-    """
+    """The acousticbrainz genre dataset"""
```

Comment:
i think the license info is missing for this dataset

Reply:
It is! Good catch

Reply:
mmmmm This dataset is composed of 4 subdatasets. Three of them are Creative Commons Attribution Non Commercial Share Alike 4.0 International and the other one is non-comercial.

>Please include in the justification field your academic affiliation (if you have one) and a brief description of your research topics and why you would like to use this dataset. If you do not include this information we may not approve your request.

The MediaEval AcousticBrainz Genre AllMusic dataset is released for non-commercial scientific research purposes only. Any publication of results based on the data extracts of the AllMusic database must cite AllMusic as the source of the data.

The MediaEval AcousticBrainz Genre AllMusic dataset is offered free of charge for internal non-commercial use only.  You may not redistribute, publically communicate or modify it, unless expressly permitted by the Universitat Pompeu Fabra (UPF) or by applicable law. The individual contents of the dataset may be protected by copyright, and a non-transferrable limited license to reproduce and use the same is granted for the indicated purpose only.
 
The dataset and its contents are made available on an “as is” basis and without warranties of any kind, including without limitation satisfactory quality and conformity, merchantability, fitness for a particular purpose, accuracy or completeness, or absence of errors. Subject to any liability that may not be excluded or limited by law, the UPF is not liable for, and expressly excludes, all liability for loss or damage however and whenever caused to anyone by any use of the MediaEval AcousticBrainz Genre AllMusic dataset or any part of it.

Reply:
Mmmmm.. ok, will try to be more specific. What about giantestps_tempo? I couldn't find the license

Reply:
I looked a bit for it and I couldn't find an explicit license as well. Looks like it's `Creative Commons Attribution Share Alike 4.0 International` like `giantsteps_key`, as they were compiled and presented together, and data is extracted from the same source.

Reply:
@PRamoneda can you take a look at the updated license of acousticbrainz and let me know if you agree?

Reply:
It is perfect for me!